cfme/tests/containers/test_table_views.py

```diff
@@ -66,8 +66,10 @@ def test_default_views(random_default_views):
 
 
 @pytest.mark.polarion('CMP-10570')
-def test_table_views():
+def test_table_views(appliance):
     for obj in objects_mapping.keys():
+        if "Collection" in obj.__name__:
```

Comment:
I would create some AbstractCollection class, inherit this in the collection classes and then check with isinstance. But anyway don't think it's critical. up to you.

Reply:
I think you are right, we spoke about doing it before, I will do it